django/db/backends/mysql/features.py

```diff
@@ -69,3 +69,9 @@ def has_zoneinfo_database(self):
 
     def introspected_boolean_field_type(self, *args, **kwargs):
         return 'IntegerField'
+
+    @cached_property
+    def can_return_last_inserted_id_with_auto_is_null(self):
```

Comment:
I guess I'd put sort this method below the other "can_*" feature (can_introspect_foreign_keys)